### PR TITLE
using environment variable for "network configmap name"

### PR DESCRIPTION
**Pull request by AjeyHiremath**

### CODE REVIEW

**Comment by AjeyHiremath** [comment]
@jkraj

File: infoblox-cni-install.yaml [hunk 1]
```diff
@@ -42,6 +43,12 @@ spec:
       containers:
       - name: install-cni
         image: infoblox/infoblox-cni-install
+		   env:
```
Comment (jkraj) [inline infoblox-cni-install.yaml]: You should correct the space here. I think tab space is used instead of normal space character for that line alone.

Comment (AjeyHiremath) [reply infoblox-cni-install.yaml]: done

File: infoblox-cni-install.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ metadata:
     app: infoblox-cni-cfg
 data:
   ## Network Config file contents##
+  key-conf-fl: 00-infoblox-ipam.conf
```
Comment (jkraj) [inline infoblox-cni-install.yaml]: conf_file_name

File: infoblox-cni-install.yaml [hunk 3]
```diff
@@ -9,6 +9,7 @@ metadata:
...
   00-infoblox-ipam.conf: |
```
Comment (jkraj) [inline infoblox-cni-install.yaml]: mention here this name should match the 'ipam_conf_file_name'

File: infoblox-cni-install.yaml (hunk 2 above)
Comment (AjeyHiremath) [reply infoblox-cni-install.yaml]: done

File: infoblox-cni-install.yaml (hunk 3 above)
Comment (AjeyHiremath) [reply infoblox-cni-install.yaml]: done

**Comment by jkraj** [comment]
@adsri Just review and give comment if any then I will approve.

**Pull request merged**